pom.xml

```diff
@@ -30,12 +30,9 @@
 	<url>https://wiki.jenkins-ci.org/display/JENKINS/Lockable+Resources+Plugin</url>
```

Comment:
This could be updated to the newer "https://wiki.jenkins.io/display/JENKINS/Lockable+Resources+Plugin".

Reply:
Right, done.